diff --git a/ReactCommon/react/runtime/hermes/HermesInstance.cpp b/ReactCommon/react/runtime/hermes/HermesInstance.cpp
index 485912d..121a80c 100644
--- a/ReactCommon/react/runtime/hermes/HermesInstance.cpp
+++ b/ReactCommon/react/runtime/hermes/HermesInstance.cpp
@@ -110,7 +110,9 @@ class HermesJSRuntime : public JSRuntime {
   }
 
   void unstable_initializeOnJsThread() override {
+#ifndef NDEBUG
     runtime_->registerForProfiling();
+#endif
   }
 
  private:
diff --git a/React/Fabric/Mounting/ComponentViews/ScrollView/RCTScrollViewComponentView.mm b/React/Fabric/Mounting/ComponentViews/ScrollView/RCTScrollViewComponentView.mm
--- a/React/Fabric/Mounting/ComponentViews/ScrollView/RCTScrollViewComponentView.mm
+++ b/React/Fabric/Mounting/ComponentViews/ScrollView/RCTScrollViewComponentView.mm
@@ -309,5 +309,4 @@
   _contentInset = contentInset;
   _scrollView.contentInset = contentInset;
-  _scrollView.scrollIndicatorInsets = contentInset;
 }
 #endif // macOS]
diff --git a/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm b/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm
--- a/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm
+++ b/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm
@@ -66,3 +66,3 @@
   NSSet<NSString *> *_Nullable _propKeysManagedByAnimated_DO_NOT_USE_THIS_IS_BROKEN;
-  RCTPlatformView *_containerView; // [macOS]
+  RCTUIView *_containerView; // [macOS]
   BOOL _useCustomContainerView;
@@ -1038,3 +1038,3 @@
     if (!_containerView) {
-      _containerView = [[RCTPlatformView alloc] initWithFrame:CGRectMake(0, 0, self.frame.size.width, self.frame.size.height)]; // [macOS]
+      _containerView = [[RCTUIView alloc] initWithFrame:CGRectMake(0, 0, self.frame.size.width, self.frame.size.height)]; // [macOS]
       for (RCTPlatformView *subview in self.subviews) { // [macOS]
